content/en/docs/zero-code/obi/setup/docker.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Run OBI as a Docker container
+menuTitle: Docker
+description:
+  Learn how to set up and run OBI as a standalone Docker container that
+  instruments another container.
+weight: 2
+---
+
+# Run OBI as a Docker container
+
+OBI can run a standalone Docker container that can instrument a process running
+in another container.
+
+Find the latest image of OBI on
+[Docker Hub](https://hub.docker.com/r/grafana/beyla) with the following name:
+
+```
+grafana/beyla:latest
+```
```

Comment:
@open-telemetry/ebpf-instrumentation-approvers 
Do we publish images to dockerhub already, and if so what are the updated names?